README.md

```diff
@@ -58,6 +58,12 @@
 
 ![](https://streampark.apache.org/image/sqlide.png)
 
+## 🚀 QuickStart
+- [Start with Docker](docker/README.md)
+- [Start with Kubernetes](helm/README.md)
```

Comment:
Is it better to put the readme here or the docker tutorial on the official website?

Reply:
> Is it better to put the readme here or the docker tutorial on the official website?

I think it's better and more intuitive to include links to the deploy project on docker (docker/README.md) directly. 😉